lib/flight_job/models/script.rb

```diff
@@ -38,13 +38,11 @@ def self.load_all(opts = nil)
         id = File.basename(File.dirname(path))
         script = new(id: id)
         if script.pass_filter?(opts)
-          if script.valid?(:load)
-            script
-          else
-            FlightJob.logger.error("Failed to load missing/invalid script: #{id}")
+          unless script.valid?(:load)
+            FlightJob.logger.error("Invalid script: #{id}")
```

Comment:
This should be logged at warning rather than error.  As we're handling this case much better than before, error now seems excessive.